### DISCUSSION TITLE
V1.1.0

### DISCUSSION THREAD

**Discussion by NadavShaar** [NadavShaar/react-grid-table#104] [Announcements]
- support for react 17.0.1 or newer as a peer dependency.
- Fixed issues: #91 #93.
- Added: rowsApi.originalRows - the rows data untouched (in sync mode - the rows data before the search filter)
- Added: searchApi.validSearchText - is an empty string if the searched text did not pass the `minSearchChars`, if it does pass, it will be equal to `searchText`
- asyncApi.resetRows - Will now also reset the selected rows and edit row
- paginationApi - Added logic to automatically set the page to be within the range of possible pages
- tableManager - Added logic to filter out selected rows that do not exist in the data anymore
- tableManager - Added logic to reset the edit row if it does no exist in the data anymore
- UI - Replaced the clear selection icon in the footer.

<hr /><em>This discussion was created from the release <a href='https://github.com/NadavShaar/react-grid-table/releases/tag/'>V1.1.0</a>.</em>

**Comment by bashintei** [comment]
Is there a limit to the number of tables that can be displayed on one page?
